components/supportivekoala/supportivekoala.app.mjs

```diff
@@ -1,11 +1,77 @@
+import { axios } from "@pipedream/platform";
+
 export default {
   type: "app",
   app: "supportivekoala",
-  propDefinitions: {},
+  propDefinitions: {
+    templateId: {
+      type: "string",
+      label: "Template ID",
+      description: "Template that defines how the image will be created",
+      async options() {
+        const templates = await this.getTemplates();
+        return templates.map((template) => ({
+          label: template.name,
+          value: template._id,
+        }));
+      },
+    },
+  },
   methods: {
-    // this.$auth contains connected account data
-    authKeys() {
-      console.log(Object.keys(this.$auth));
+    async _makeRequest({
+      $ = this,
+      opts,
+    }) {
+      const {
+        method = "get",
+        path,
+        params,
+        data,
+      } = opts;
+      return axios($, {
+        headers: {
+          "Authorization": `Bearer ${this.$auth.api_key}`,
+          "Content-Type": "application/json",
+        },
+        url: `https://api.supportivekoala.com/v1${path}`,
+        method,
+        params,
+        data,
+      });
+    },
```

Comment:
You can allso do:
```suggestion
    async _makeRequest({
      $ = this, path, ...args
    } = {}) {
      return axios($, {
        headers: {
          "Authorization": `Bearer ${this.$auth.api_key}`,
          "Content-Type": "application/json",
        },
        url: `https://api.supportivekoala.com/v1${path}`,
        ...args
      });
    },
```
If you don't pass `method` then `axios` lib by default will set it for you as `GET`